procedures/MT/rotator/rotator_verification.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<p>Consider:</p><p>The test conditions are assigned by Tekniker and parameters have been hard-coded in functions to simplify the functional call. &nbsp;The details can be seen in the <a href="https://github.com/lsst-ts/ts_mt_hexRot_simulink/blob/develop/doc/TestPlan.pdf" target="_blank">test plan</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Fixed in commit: [da2822c](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/da2822c02b133ed099461c12c208f882ed94bd6d). Thanks!